packages/truffle-artifactor/test/methods.js

```diff
@@ -0,0 +1,40 @@
+const assert = require("assert");
+const Artifactor = require("../");
+
+describe("Artifactor.save", () => {
+  it("throws if passed an artifact without a contractName", () => {
+    artifactor = new Artifactor("/some/path");
+
+    artifactor
+      .save({
+        "abi": [],
+        "bytecode": "0xabcdef",
+        "networks": {
+          3: { address: "0xe6e1652a0397e078f434d6dda181b218cfd42e01" }
+        },
+        "x-from-dependency": "somedep"
```

Comment:
You might also want to include a `then` that contains something like `assert(false)` to guarantee that it fails. Otherwise, if it doesn't throw an error it is making no assertion.